### DISCUSSION TITLE
a few minor theming questions from a new user

### DISCUSSION THREAD

**Discussion by ExpandingMan** [nushell/nushell#13794] [Q&A] [answered]
There are a few things I haven't been able to track down in the documentation.  Which are the `color_config` values for
- Built-in commands (always seem bold cyan no matter what I set)
- Selection highlight (always opaque white color preventing me from seeing highlighted text at all)

Thanks!

**Comment by fdncred** [comment]
selection highlight is not currently exposed via config. the repl syntax highlighting is controlled by the shapes in the theme. i believe built-in commands are controlled by shape_internalcall.

**Comment by ExpandingMan** [comment]
I thought it would be `shape_internalcall`, but that doesn't seem to do anything.

Is there nothing I can do about the selection highlight?  It makes the selection unreadable so it's a bit of a problem.  I suppose it may be my terminal config, but it looks fine in other shells.

**Reply by fdncred** [reply, marked as answer]
I tested it out before I responded to make sure I was saying the right shape. It works.

No, there's nothing you can do about selection highlighting. It's a reedline issue. There's a PR for it in that repo I think.

**Reply by ExpandingMan** [reply]
Oh, there was a typo in my key name, thanks.

**Comment by ExpandingMan** [comment]
On a related note, I also can't figure out how to set the colors for the arguments of built in functions.  They look `green` for most functions such as `ls` but `cyan` (same as command, though command is bold) for `cd`.

**Reply by fdncred** [reply]
Shapes control coloring in the repl. Paths parameters will be a different color from string parameters from glob parameters.
